lua/autorun/server/toggle_voice_init.lua

```diff
@@ -44,13 +45,8 @@ hook.Add( "PlayerCanHearPlayersVoice", "CFC_ToggleLocalVoice_CanHear", function(
 end)
 
 
-concommand.Add("enable_proximity_voice", function(ply)
-    ply:ChatPrint( "Proximity voice enabled!")
-    playerConfig[ply] = true
-end)
-
-concommand.Add("disable_proximity_voice", function(ply)
-    ply:ChatPrint( "Proximity voice disabled!")
-    playerConfig[ply] = nil
-end)
-
+util.AddNetworkString( "proximity_voice_changed" )
+net.Receive( "proximity_voice_changed", function( len, ply )
+    local enabled = net.ReadBool()
+    playerConfig[ply] = enabled or nil
```

Comment:
Could you add some playerdisconnected hooks to make sure this table doesn't balloon up